packages/@aws-cdk-testing/alpha-module-stable-tests/README.md

```diff
@@ -0,0 +1,31 @@
+# CDK CLI integration test
```

Comment:
Does this package need to exist? They look like unit tests only, any particular reason these tests couldn't just go into the alpha module itself (or the alpha integration module itself)? 

It already depends on `aws-cdk-lib` anyway, so putting unit tests there wouldn't add a dependency arrow.

Reply:
these are unit tests, and they are unit tests that depend on both `aws-cdk-lib` and an alpha module (here, `aws-batch`). They can't go in `aws-cdk-lib`, because then `aws-cdk-lib` depends on `aws-batch`, and that is a circular dependency that Lerna yells at me for:

```lerna ERR! ECYCLE Dependency cycles detected, you should fix these!
lerna ERR! ECYCLE aws-cdk-lib -> @aws-cdk/aws-batch-alpha -> @aws-cdk/integ-runner -> aws-cdk -> aws-cdk-lib
```

We could put them in the alpha module itself. Is that preferable?